example/App.tsx

```diff
@@ -56,11 +56,34 @@ export default class App extends React.Component<IAppProps, IAppState> {
                 <Button
                     title='cancel Job'
                     onPress={() => {
-                      if(this.state.jobId){
-                        queue.cancelJob(this.state.jobId, {message: 'Canceled'})
-                      } else {
-                        console.log("no job running");
-                      }
+                        if (this.state.jobId) {
+                            queue.cancelJob(this.state.jobId, { message: 'Canceled' })
+                        } else {
+                            console.log("no job running");
+                        }
+                    }}
+                />
+                <Button
+                    title='remove failed Jobs'
+                    onPress={async () => {
+                        let jobs = await queue.getJobs();
+                        let jobRemovals = jobs.filter(job => job.failed !== '').map(async job => await queue.removeJob(job))
+                        await Promise.all(jobRemovals)
+                    }}
+                />
+                <Button
+                    title='requeue failed Jobs'
+                    onPress={async () => {
+                        let jobs = await queue.getJobs();
+                        let jobRequeues = jobs.filter(job => job.failed !== '').map(async job => await queue.requeueJob(job))
+                        await Promise.all(jobRequeues)
+                    }}
+                />
+                <Button
+                    title='getJobs'
+                    onPress={async () => {
+                        let jobs = await queue.getJobs();
```

Comment:
Seeing this in a long run.
What if i have 1000s of jobs in store.
Would it be a good idea to retrieve all sometimes?

Maybe we could add options to limit the records.

Reply:
Yes i think this is a good idea, but this should be addressed in an another PR